docker/Dockerfile

```diff
@@ -28,7 +28,7 @@ ENV PGRST_DB_URI= \
     PGRST_DB_SCHEMA=public \
     PGRST_DB_ANON_ROLE= \
     PGRST_DB_POOL=100 \
-    PGRST_SERVER_HOST=*4 \
+    PGRST_SERVER_HOST="127.0.0.1" \
```

Comment:
this is not a good default for docker (it should stay *4). This will mean if you start postgrest container it will listen only internally and you will not be able to actually work with it.

Reply:
I overlooked that, though I believe that's only true for macos/windows that run the docker host in a vm, in linux it works fine. I'll make the change.

Reply:
i don't think it's about the vm. have you actually built the image? (i don't think you could since the build process downloads the binary from the release). 
regardless of the os, in docker each container gets assigned an external ip/interface, that is the interface for which docker bridges the external/internal port, this (127...) will not work on any os (but yes, it failed me on mac)

Reply:
It can be tested by overriding the `PGRST_SERVER_HOST`(not necessary to build the image), on linux it works with `127.0.0.1`.

Reply:
Used this https://gist.github.com/steve-chavez/77d5ef4e8b0a51120bc020b96d6553f5.

Reply:
that gist works only because you have `network_mode: host` there, which is not the standard way of running docker containers, if you remove that line you'll see the problem